modules/reference/pages/systems/system-details/sd-virtualization.adoc

```diff
@@ -1,310 +1,94 @@
-
-[guimenu]``Adapter name``::
-SCSI adapter name for ``scsi_host`` controller.
-
-[guimenu]``Adapter parent PCI address``::
-PCI address of the SCSI host in ``0000:00:00.0`` format.
-List options with [command]``lsscsi -v``.
-
-[guimenu]``Adapter parent address unique ID``::
-Unique ID of the SCSI host as found in ``/sys/class/scsi_host/host*/unique_id`` file.
-
-[guimenu]``Adapter parent name``::
-Name of the vport capable parent SCSI host of the virtual Host Bus Adapter (vHBA).
-
-[guimenu]``Adapter parent wwnn``::
-World Wide Node Name used by the ``fc_host`` to identify the vHBA parent device.
-
-[guimenu]``Adapter parent wwpn``::
-World Wide Port Name used by the ``fc_host`` to identify the vHBA parent device.
-
-[guimenu]``Adapter parent fabric wwn``::
-Fabric WWN of the vHBA parent device.
-
-[guimenu]``Adapter wwnn``::
-World Wide Node Name used by the ``fc_host`` to identify the vHBA device.
-
-[guimenu]``Adapter wwpn``::
-World Wide Port Name used by the ``fc_host`` to identify the vHBA device.
-
-[guimenu]``Manage vHBA deletion``::
-If checked the vHBA will be destroyed with the pool is destroyed.
-This property will be automatically activated if there is no existing vHBA.
-
-==== Target fields
-
-[guimenu]``Path``::
-Path to the storage pool mount or device on the virtual host.
-
-[guimenu]``Owner ID``::
-ID of the user owning the path folder or file.
-
-[guimenu]``Group ID``::
-ID of the group owning the path folder or file.
-
-[guimenu]``Permission mode``::
-Octal representation of the permissions to set on the path folder or file.
 
-[guimenu]``SELinux label``::
-SELinux label to set on the path folder or file.
 
-=== Editing a pool
+== Storage
 
-To edit the properties of a storage pool, locate the pool in the list and click [guimenu]``Edit pool``.
+This section lists the virtual storage pools and volumes defined on the selected client.
+This is only available when the selected client is registered as a Salt virtual host.
 
-=== Deleting a Pool
+The list shows these columns:
 
-To delete a storage pool, locate the pool in the list and click [guimenu]``Delete``.
-By default, deleting a pool only removes the storage pool definition.
-The pool data is kept on disk.
-To delete the pool data as well as the storage pool definition, check the [guimenu]``Delete the pool, including the contained volumes`` box before you click [guimenu]``Delete``.
+[[virt-pool-columns]]
+[cols="1,1", options="header"]
+.Virtual Storage Pools List Columns
+|===
+| Column            | Description
+| Name              | The name of the virtual storage pool or volume
+| State             | Indicates if the pool is running, suspended, stopped, or crashed
+| Autostart         | Checked if the pool starts automatically when the virtual host is booted
+| Persistent        | Checked if the pool persists between virtual host reboots
+| Location          | The absolute path to the pool on the virtual host
+| Usage             | The percentage of storage in use.
```

Comment:
here we probably have fulstops because there are 2 sentences.

Reply:
Yes, that's right.